src/modules/fw_lateral_longitudinal_control/FwLateralLongitudinalControl.cpp

```diff
@@ -0,0 +1,764 @@
+	const Vector2f heading_vector(matrix::Dcm2f(local_pos.heading)*Vector2f({1.f, 0.f}));
+	const Vector2f ground_vel_norm(ground_vel.normalized());
+	const float flying_forward_factor(math::constrain((heading_vector.dot(ground_vel_norm) -
+							   COS_HEADING_TRACK_ANGLE_PUSHED_BACK) / ((COS_HEADING_TRACK_ANGLE_NOT_PUSHED_BACK -
+												    COS_HEADING_TRACK_ANGLE_PUSHED_BACK)),
+							  0.f, 1.f));
+
+	return flying_forward_factor * low_ground_speed_factor;
+}
+float FwLateralLongitudinalControl::getCorrectedLateralAccelSetpoint(float lateral_accel_sp)
+{
+	// Scale the npfg output to zero if npfg is not certain for correct output
+	const float can_run_factor{math::constrain(getGuidanceQualityFactor(_local_pos, _wind_valid), 0.f, 1.f)};
+
+	hrt_abstime now{hrt_absolute_time()};
+
+	// Warn the user when the scale is less than 90% for at least 2 seconds (disable in transition)
+
+	// If the npfg was not running before, reset the user warning variables.
+	if ((now - _time_since_last_npfg_call) > ROLL_WARNING_TIMEOUT) {
+		_need_report_npfg_uncertain_condition = true;
+		_time_since_first_reduced_roll = 0U;
+	}
+
+	if (_vehicle_status_sub.get().in_transition_mode || can_run_factor > ROLL_WARNING_CAN_RUN_THRESHOLD || _landed) {
+		// NPFG reports a good condition or we are in transition, reset the user warning variables.
+		_need_report_npfg_uncertain_condition = true;
+		_time_since_first_reduced_roll = 0U;
+
+	} else if (_need_report_npfg_uncertain_condition) {
+		if (_time_since_first_reduced_roll == 0U) {
+			_time_since_first_reduced_roll = now;
+		}
+
+		if ((now - _time_since_first_reduced_roll) > ROLL_WARNING_TIMEOUT) {
+			_need_report_npfg_uncertain_condition = false;
+			events::send(events::ID("npfg_roll_command_uncertain"), events::Log::Warning,
+				     "Roll command reduced due to uncertain velocity/wind estimates!");
+		}
+
+	} else {
+		// Nothing to do, already reported.
+	}
+
+	_time_since_last_npfg_call = now;
+
+	return can_run_factor * (lateral_accel_sp);
+}
+float FwLateralLongitudinalControl::mapLateralAccelerationToRollAngle(float lateral_acceleration_sp) const {
+	return  atanf(lateral_acceleration_sp * 1.0f / CONSTANTS_ONE_G);
+}
+
+void FwLateralLongitudinalControl::setDefaultLongitudinalControlLimits() {
+		_long_limits.timestamp = hrt_absolute_time();
+		_long_limits.equivalent_airspeed_min = _performance_model.getMinimumCalibratedAirspeed();
+		_long_limits.equivalent_airspeed_max = _performance_model.getMaximumCalibratedAirspeed();
+		_long_limits.pitch_min = radians(_param_fw_p_lim_min.get());
+		_long_limits.pitch_max = radians(_param_fw_p_lim_max.get());
+		_long_limits.throttle_min = _param_fw_thr_min.get();
+		_long_limits.throttle_max = _param_fw_thr_max.get();
+		_long_limits.climb_rate_target = _param_climbrate_target.get();
+		_long_limits.sink_rate_target = _param_sinkrate_target.get();
+		_long_limits.disable_underspeed_protection = false;
+		_long_limits.enforce_low_height_condition = false;
```

Comment:
```suggestion
		_long_limits.timestamp = hrt_absolute_time();
		_long_limits.equivalent_airspeed_min = _performance_model.getMinimumCalibratedAirspeed();
		_long_limits.equivalent_airspeed_max = _performance_model.getMaximumCalibratedAirspeed();
		_long_limits.pitch_min = radians(_param_fw_p_lim_min.get());
		_long_limits.pitch_max = radians(_param_fw_p_lim_max.get());
		_long_limits.throttle_min = _param_fw_thr_min.get();
		_long_limits.throttle_max = _param_fw_thr_max.get();
		_long_limits.climb_rate_target = _param_climbrate_target.get();
		_long_limits.sink_rate_target = _param_sinkrate_target.get();
		_long_limits.disable_underspeed_protection = false;
		_long_limits.enforce_low_height_condition = false;
```
The rows has 2 indents